src/main/java/nl/tudelft/ti2206/group9/util/Resource.java

```diff
@@ -0,0 +1,163 @@
+package nl.tudelft.ti2206.group9.util;
+
+import static nl.tudelft.ti2206.group9.util.GameObservable.OBSERVABLE;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Enumeration;
+import java.util.List;
+import java.util.jar.JarEntry;
+import java.util.jar.JarFile;
+
+import nl.tudelft.ti2206.group9.util.GameObserver.Category;
+import nl.tudelft.ti2206.group9.util.GameObserver.Error;
+
+/**
+ * Helper class to quickly load resource URLs, URI strings, etc.
+ * @author Maarten
+ */
+public final class Resource {
+
+    /** Hiding public constructor. */
+    private Resource() { }
+
+    /**
+     * @param packagePath The package path of the resource.
+     * @return the URI.toURL() of this resource.
+     */
+    public static URL getURL(final String packagePath) {
+        final URL res = Thread.currentThread().getContextClassLoader()
+                .getResource(packagePath);
+        if (res == null) {
+            OBSERVABLE.notify(Category.ERROR, Error.RESOURCEEXCEPTION,
+                    "Resource.getURL(String)", "getResource() == null",
+                    packagePath);
+        }
+        return res;
+    }
+
+    /**
+     * @param packagePath The package path of the resource.
+     * @return the URI of this resource.
+     */
+    public static URI getURI(final String packagePath) {
+        try {
+            return getURL(packagePath).toURI();
+        } catch (URISyntaxException e) {
+            OBSERVABLE.notify(Category.ERROR, Error.RESOURCEEXCEPTION,
+                    "Resource.getURI(String)", e.getMessage(), packagePath);
+        }
+        return null;
+    }
+
+    /**
+     * @param packagePath The package path of the resource.
+     * @return the URI.toString() of this resource.
+     */
+    public static String getURIString(final String packagePath) {
+        return getURI(packagePath).toString();
+    }
+
+    /**
+     * @param packagePath The package path of the resource.
+     * @return the URL.toString() of this resource.
+     */
+    public static String getURLString(final String packagePath) {
```

Comment:
Method is never used according to my IDE.
Is this true? If not, please remove.

< review, logic